src/vm/sbin/vmadmd.js

```diff
@@ -775,7 +853,8 @@ function updateZoneStatus(ev)
             'internal_metadata',
             'state',
             'uuid',
-            'zone_state'
+            'zone_state',
+            'zonepath'
         ]}, function (err, vmobj) {
             log.info(ev.zonename + ' (docker) went from ' + ev.oldstate + ' to '
```

Comment:
We should probably at least log. At this point though, updateZoneStatus() doesn't call a callback, so it's not really clear what we should do when a VM.load fails for a single update. I'll file a ticket to look into whether we can do some sort of retry here.